libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  AfterContentInit,
+  AfterViewInit,
+  ContentChild,
+  ContentChildren,
+  EventEmitter,
+  Output,
+  QueryList,
+  TemplateRef,
+  ViewChildren,
+} from '@angular/core';
+import { Component, Input } from '@angular/core';
+
+import { ListItemTemplateDirective } from '../../list/list.directive';
+import { RadioComponent } from '../radio.component';
+
+@Component({
+  selector: 'kirby-radio-group',
+  templateUrl: './radio-group.component.html',
+  styles: ['ion-radio-group { display: inherit; flex-wrap: inherit}'],
+})
+export class RadioGroupComponent implements AfterContentInit, AfterViewInit {
+  @ContentChild(ListItemTemplateDirective, { static: true, read: TemplateRef })
+  itemTemplate: TemplateRef<any>;
+
+  @ViewChildren(RadioComponent)
+  private radioButtons: QueryList<RadioComponent>;
+
+  @ContentChildren(RadioComponent, { descendants: true })
+  private slottedRadioButtons: QueryList<RadioComponent>;
+
+  private _items: string[] | any[] = [];
+  get items(): string[] | any[] {
+    return this._items || []; // Ensure items return empty array even if set to null/undefined
+  }
+
+  @Input() set items(items: string[] | any[]) {
```

Comment:
WARNING: Highly opinionated ;) 
I would love having the input and outputs at the top, to make it easy to get a quick overview of the component. I see what you doing, grouping it by context, but then I need to read the whole component or trust the documentation to get an overview of the component API

Reply:
Done 👍